infrastructure/transform/terragrunt.hcl

```diff
@@ -100,6 +136,13 @@ provider "databricks" {
   azure_workspace_resource_id = azurerm_databricks_workspace.databricks.id
   host                        = azurerm_databricks_workspace.databricks.workspace_url
 }
+
+provider "databricks" {
+  host = "https://accounts.azuredatabricks.net"
+  alias = "accounts"
+  account_id = "e5d77844-84a5-45ea-aa80-21ea9743d234"
```

Comment:
@jjgriff93  could you help me find a way to pass this in (but make it optional)?